src/SFA.DAS.Support.Portal.Web/Views/Shared/_GoogleAnalyticsHeadPartial.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿<!-- Google Tag Manager -->
+
+<script>
```

Comment:
That's the neatest version of this script I've ever seen 😄. We could use the more minified version from [here](https://developers.google.com/tag-manager/quickstart), but it's all present and correct!